README.md

```
@@ -1,59 +1,97 @@
+- 🔄 Real-time data updates
+- 📋 Copy functionality with tooltips
+- 🏷️ Status badges for log states
 
 ## Tech Stack
 
 - **Framework**: Next.js 15.3.2
 - **Language**: TypeScript
 - **Styling**: Tailwind CSS 4
-- **UI Components**: Shadcn
-- **Authentication**: Centralised Auth app integration
+- **UI Components**: 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)